project/AppDependencies.scala

```diff
@@ -3,19 +3,24 @@ import sbt.{ModuleID, _}
 
 object AppDependencies {
 
+  val silencerVersion = "1.7.1"
+
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc" %% "bootstrap-play-26" % "2.3.0",
-    "uk.gov.hmrc" %% "play-ui" % "8.19.0-play-26"
+    "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "5.3.0",
+    "uk.gov.hmrc" %% "play-ui" % "9.2.0-play-27",
```

Comment:
I feel like we don't need play-ui as an api?

Reply:
frankly I forgot to check, I kinda assumed it's using some helper methods from it or some such but I just tested and it indeed is unneeded - will toss it out